system/inc/system_dynalib.h

```diff
@@ -102,7 +102,8 @@ DYNALIB_FN(BASE_IDX + 16, system, system_invoke_event_handler, int(uint16_t hand
 
 #if HAL_PLATFORM_POWER_MANAGEMENT
 DYNALIB_FN(BASE_IDX + 17, system, system_power_management_set_config, int(const hal_power_config*, void*))
-#define BASE_IDX1 (BASE_IDX + 18)
+DYNALIB_FN(BASE_IDX + 18, system, system_power_management_get_config, int(hal_power_config*, void*))
+#define BASE_IDX1 (BASE_IDX + 19)
```

Comment:
This breaks the dynalib table, which will cause incompatibility. We should keep `BASE_IDX1` and append the dynalib function at the end of the table. So it should be:
```
#if HAL_PLATFORM_POWER_MANAGEMENT
DYNALIB_FN(BASE_IDX1 + 2, system, system_power_management_get_config, int(hal_power_config*, void*))
#define BASE_IDX2 (BASE_IDX1 + 3)
#else
#define BASE_IDX2 (BASE_IDX1 + 2)
#endif // HAL_PLATFORM_POWER_MANAGEMENT
```
following the `DYNALIB_FN(BASE_IDX1 + 1, system, system_reset, int(unsigned, unsigned, unsigned, unsigned, void*))`.

Reply:
Good catch!  I thought it was already at the end